pkg/operator/resources/namespaced/controller.go

```diff
@@ -175,7 +175,7 @@ func createControllerServiceAccount() *corev1.ServiceAccount {
 
 func createControllerDeployment(controllerImage, importerImage, clonerImage, uploadServerImage, verbosity, pullPolicy string, imagePullSecrets []corev1.LocalObjectReference, priorityClassName string, infraNodePlacement *sdkapi.NodePlacement, replicas int32) *appsv1.Deployment {
 	defaultMode := corev1.ConfigMapVolumeSourceDefaultMode
-	deployment := utils.CreateDeployment(controllerResourceName, cdiLabel, controllerResourceName, common.ControllerServiceAccountName, imagePullSecrets, int32(1), infraNodePlacement)
+	deployment := utils.CreateDeployment(controllerResourceName, "app", "containerized-data-importer", common.ControllerServiceAccountName, imagePullSecrets, int32(1), infraNodePlacement)
```

Comment:
@awels For update success, I unchange this code. While I think use label `cdi.kubevirt.io: cdi-deployment` for pod affinity is more reasonable, so I add a the label after a few line.

Reply:
Sure, I agree for consistency it is better. Did you also see Michaels comments on the API. how you can merge the fields very similar to what you wanted originally, but keep upgrade compatibility?